app/controllers/Puzzle.scala

```diff
@@ -156,6 +157,30 @@ final class Puzzle(env: Env, apiC: => Api) extends LilaController(env):
       )
   }
 
+  def apiBatchVoteThemes = SecuredScopedBody(_.PuzzleCurator)(_.Puzzle.Write) { _ ?=> me ?=>
+    bindForm(env.puzzle.forms.batchVotes)(
+      jsonFormError,
+      _.votes
+        .sequentially(puzzleVotes =>
+          puzzleVotes.themes
+            .sequentially: themeVote =>
+              allow:
+                env.puzzle.api.theme
+                  .vote(puzzleVotes.puzzleId, themeVote.theme, themeVote.vote)
+                  .inject(none)
+              .rescue: err =>
+                fuccess(Json.obj("theme" -> themeVote.theme, "msg" -> err.message).some)
+            .map:
+              _.flatten.map: errors =>
+                Json.obj("puzzleId" -> puzzleVotes.puzzleId, "errors" -> errors)
```

Comment:
I think this is different behaviour than before simplifying?

It used to be 
```
[{"puzzleId -> "foo", "errors" -> [
     { "theme": "bar", msg: "baz"},
     { "theme": "quux", msg: "boy"}
]},
...]
```

and now will be
```
[{"puzzleId -> "foo", "errors" -> { "theme": "bar", msg: "baz"},
 {"puzzleId -> "foo", "errors" -> { "theme": "quux", msg: "boy"},
  ...
]
```

not a big deal though, although `errors` plural is incorrect